vignettes/dummies_guide.Rmd

```diff
@@ -0,0 +1,137 @@
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+## Introduction
+
+This vignette is to provide guidance for people who are completely unfamiliar with Open Trip Planner and/or R.  It assumes no prior-knowledge.  If you are familiar with R and/or OTP then you may find the Getting Started vignette more useful.
+
+## What is this package for, and what is a package anyway?
+
+The Open Trip Planner for R package makes it easier for R and Open Trip Planner to communicate.
+
+### What is Open Trip Planner?
+
+OpenTripPlanner (OTP) is a free, open-source, and cross-platform multi-modal route planner written in JAVA.  It is like having your own private version of Google Maps.
+
+You can find a lot more information about OTP at:
+
+* http://www.opentripplanner.org/
+* https://wiki.openstreetmap.org/wiki/OpenTripPlanner
+
+### What is R?
+
+R is a programming language and free software environment for statistical computing and graphics.  R has many capabilities for analysing data and writing software, but in this context, its ability to produce and analyse spatial data and maps is most relevant.
+
+You can find a lot more information about R at:
+
+* https://www.r-project.org/ 
+* https://en.wikipedia.org/wiki/R_(programming_language) 
+
+### What is an R package?
+
+An R package is a small piece of software that extends the basic capabilities of R. It is a bit like how a new phone can do some things out of the box (make phone calls, send email) but you have to install apps to add extra abilities.
+
+### Why would I want to use Open Trip Planner for R?
+
+OTP can be used to run public route finding services such as https://ride.trimet.org but it can also be run on your own private computer or server. If you want, analyse a transport network OTP is a very useful and tool. However, OTP is, as the name suggests, a Trip Planner, not analytical software.  So, while OTP can find the fastest route from A to B or find all the places that are within a 20-minute walk of C, it cannot answer a question like “how many people live within 10-minutes of a park?”  This is where R can help.  R can process multiple spatial datasets such a population densities and park locations, but does not have a built-in journey planner. 
+
+This package allows you to use the trip planning power of OTP with the analytical might of R.
+
+## What do I need to get started?
+
+You will need to install some software to use OTP and R.
+
+### To use OTP
+
+* Download free Java 8 from https://www.java.com/en/download/ and install Java. You will need the correct version for your operating system.
+* Download the .shaded.jar OTP  file from https://repo1.maven.org/maven2/org/opentripplanner/otp/ at time of writing the latest version of OTP is version 1.3 so you would want to get https://repo1.maven.org/maven2/org/opentripplanner/otp/1.3.0/otp-1.3.0-shaded.jar 
+
```

Comment:
@mem48 Probably need a word of warning here. The release version of OTP is not well-suited for all counries (it is tuned for USA). The OTP.JAR in the tutorial is a modified version where I change edge traversal permissions and adjusted speed limits to better match the UK situation. I think that this may now be possible to configure via the new Way property sets that can be configured and are then read at graph build. I will investigate further. This info can then be incorporated into the advanced guide (probably?).